docs/elements/hx-panelbody/index.html

```diff
@@ -0,0 +1,14 @@
+---
+title: <hx-panelbody>
+---
+{% extends 'element.njk' %}
+{% block content %}
+  <section>
+    <p>
+       An <code>hx-panelbody</code> fills the space between an optional 
+       <a href="elements/hx-panelhead">&lt;hx-panelhead&gt;</a> and
+       <a href="elements/hx-panelfoot">&lt;hx-panelfoot&gt;</a>, scrolling
```

Comment:
Is the footer optional, as well?

Reply:
yes -- added in "an optional" in front of footer